specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/controller.json

```diff
@@ -70,7 +70,7 @@
           "default": {
             "description": "Error response describing why the operation failed.",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/definitions/ControllerErrorResponse"
```

Comment:
Please do not change the errorresponse name as it is SDK breaking change. 

In fact, please update ALL your swagger to reference the one under common-types\resource-management\v2\types.json\ErrorResponse. 

You should also remove all standard parameter definitions (subscription, ApiVersion, RGName) and reference the standard versions.